frontend/src/components/pages/StatisticsPage.tsx

```diff
@@ -0,0 +1,461 @@
+      </Box>
+
+      <Box display="flex" gap={2} flexWrap="wrap" mb={3}>
+        <TextField
+          label="Название содержит"
+          value={filters.title}
+          onChange={e => setFilters(f => ({ ...f, title: e.target.value }))}
+          size="small"
+        />
+        <FormControl size="small" style={{ minWidth: 140 }}>
+          <InputLabel>{attrToRussian['category']}</InputLabel>
+          <Select
+            value={filters.category}
+            label={attrToRussian['category']}
+            onChange={e => setFilters(f => ({ ...f, category: e.target.value }))}
+          >
+            <MenuItem value="">Все</MenuItem>
+            {unique('category').map(val => (
+              <MenuItem key={val} value={val}>{categoryToRussian(val as any) ?? val}</MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+        <Autocomplete
+          options={unique('address')}
+          value={filters.address}
+          onInputChange={(_, newValue) => setFilters(f => ({ ...f, address: newValue }))}
+          onChange={(_, newValue) => setFilters(f => ({ ...f, address: newValue ?? '' }))}
+          renderInput={(params) => (
+            <TextField
+              {...params}
+              label={attrToRussian['address']}
+              size="small"
+              placeholder="Поиск или выберите"
+            />
+          )}
+          freeSolo
+          clearOnEscape
+          sx={{ minWidth: 280, maxWidth: 400 }}
+        />
+        <Autocomplete
+          options={userIdOptions}
+          getOptionLabel={option => typeof option === 'string' ? option : option.fullname}
+          value={userIdOptions.find(opt => opt.id === filters.user_id) || null}
+          onChange={(_, newValue) => setFilters(f => ({ ...f, user_id: newValue?.id ?? '' }))}
+          renderInput={(params) => (
+            <TextField
+              {...params}
+              label={attrToRussian['user_fullname']}
+              size="small"
+              placeholder="Поиск или выберите"
+            />
+          )}
+          isOptionEqualToValue={(option, value) => {
+            if (!option || !value) return false;
+            return option.id === value.id;
+          }}
+          clearOnEscape
+          sx={{ minWidth: 250, maxWidth: 300 }}
+        />
+        <TextField
+          label="Мин. цена"
+          type="number"
+          value={filters.minPrice}
+          onChange={e => setFilters(f => ({ ...f, minPrice: e.target.value }))}
+          size="small"
+        />
+        <TextField
+          label="Макс. цена"
+          type="number"
+          value={filters.maxPrice}
+          onChange={e => setFilters(f => ({ ...f, maxPrice: e.target.value }))}
+          size="small"
+        />
+        <TextField
+          label="Мин. дата изменения"
+          type="date"
+          value={dateFrom}
+          onChange={e => setDateFrom(e.target.value)}
+          size="small"
+          InputLabelProps={{ shrink: true }}
```

Comment:
Лучше использовать `slotProps.inputLabel`:

```suggestion
          slotProps={{ inputLabel: { shrink: true }}}
```